### ISSUE TITLE
Add ability to find polls

### ISSUE THREAD

**Issue by Yakaita** [guilhermeg2k/Anipool#6]
Maybe a button to go to a random poll thats still active. or just a way to see other people polls without being directly linked. 

if you dont have a large group of people to see you poll then it wont get any votes

**Comment by guilhermeg2k** [comment]
Yes this is one of the features that i already mapped to add in the future, here you can take a look what i've been looking to implement further https://anipool.notion.site/c717de9fa8ac4616ab9af92e28f41b32?v=21bed3e8e8954820b63e110876f3a8d7
I'll update that issue when got my hands to work on it
